python/requirements/ml/data-test-requirements.txt

```diff
@@ -1,11 +1,13 @@
 # Used by CI for datasets tests.
 # https://github.com/ray-project/ray/pull/29448#discussion_r1006256498
 
-python-snappy
-tensorflow-datasets
+python-snappy; python_version <= '3.10'
```

Comment:
is the python upper bound required here?

Reply:
good catch, I'll dig deeper in the subsequent PR where I add python 3.11 (and potentially need to fixes test failure fallout from that); i'll keep it pinned for <3.11 for now